.werft/util/tracing.ts

```diff
@@ -0,0 +1,46 @@
+import { Metadata, credentials } from "@grpc/grpc-js";
+import { NodeSDK } from '@opentelemetry/sdk-node';
```

Comment:
Could we move this file under `.werft/observability` folder? I'm suggesting that because we're the owners of that folder and we can approve our code there.

If we leave this under `.werft/util` then we can get some random undesired assignments

Reply:
Yep I can certainly do that ☺️